### PR TITLE
addressing EPSG code change to 5070

### PR DESCRIPTION
**Pull request by dylanbeaudette**
Addressing #205

### CODE REVIEW

**Comment by dylanbeaudette** [comment]
Any reasons not to finish this conversion to EPSG:5070 ?
@smroecker @brownag @hammerly @jskovlin

**Comment by smroecker** [comment]
Looks fine to me

File: NEWS.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# soilDB 2.6.5 (2021-08-18)
```
Comment (brownag) [inline NEWS.md:1]: This is a significant change that warrants the version bump, we should plan for a CRAN release to replace 2.6.3 ASAP.

**Review by brownag — APPROVED** [review]

File: NEWS.md (hunk 1 above)
Comment (dylanbeaudette) [reply NEWS.md:1]: Good idea.

**Pull request merged**